### DISCUSSION TITLE
Treat seeding in the same way as migrations

### DISCUSSION THREAD

**Discussion by mkosir** [prisma/prisma#14093] [Q&A]
Hey all,
happy user of Prisma here 🚀  
I believe it would be better if seeding would behave the same way as when we run migration where after being run they are stored in `_prisma_migration` table.
In this case when we re-run seeding, only new seeds (scripts) are being inserted into database, instead of duplicating them or failing unique constraints.

**Comment by ruheni** [comment]
Hey 👋🏽

Would you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) outlining your use case here? That would be helpful!

**Reply by mkosir** [reply]
Hey @ruheni, thx for pointing it out, [done](https://github.com/prisma/prisma/issues/14194) 👍

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
